### PR TITLE
Update ESA algorithm to use conda

### LINKED ISSUE #9: Algorithm produces different data outputs between NASA and ESA
Due to #7, it was discovered that, given the same inputs, there are discrepancies between the results produced by running the algorithm in the NASA and ESA environments.

In addition to using this algorithm to gather profiling metrics from the 2 environments, we also want to make sure we can produce the same results.

See attached slides detailing @nemo794's initial investigation into the output discrepancies: [NASA-ESA get-dem discrepancy.pptx](https://github.com/MAAP-Project/get-dem/files/15079836/NASA-ESA.get-dem.discrepancy.pptx)

### PR DESCRIPTION
**Pull request by chuckwondo**
Fixes #9

### CODE REVIEW

File: get_dem.py
Comment (nemo794) [inline get_dem.py]: Because we've already started benchmarking with the `np.pinv()` function, I don't want to derail that process and have to redo the benchmarking.

That said, having this be `np.inv()` is a quick litmus test for whether the bug is resolved or not, so I can see why it's useful to have in this PR.

These seem to be two possible paths forward:
1) Leave as `np.inv()` in this PR, but do not merge this PR until after this round of benchmarking is complete. (maybe 1-3 weeks delay?)
2) Revert to `np.pinv()` in this PR so that we can merge it asap. On the plus side, this approach means that we do not need to redo the existing benchmarking results. However, the cost is that we'll need to manually hack the code to get our litmus test of whether the "fix" worked or not... which means it's very hard to test in DPS.

@arthurduf @chuckwondo  Thoughts? Preferences?

**Comment by chuckwondo** [comment]
> However, the cost is that we'll need to manually hack the code to get our litmus test of whether the "fix" worked or not... which means it's very hard to test in DPS.

I'm not sure what you mean here.

However, I'm happy to put `pinv` back in place. Whether we use `inv` or `pinv` makes no difference in determining whether or not our "fix" worked. The "fix" is verified by the addition of the test for comparing the outputs to each other. Without the "fix" (i.e., without using `conda` in the ESA Docker image), the outputs are different whether we use `inv` or `pinv` (assuming a bbox is chosen to avoid the singularity error). As soon as `conda` is used in both places, using `inv` produces identical results, and so does `pinv`.

I'm in favor of putting `pinv` back in place for this PR.

**Comment by chuckwondo** [comment]
I put `pinv` back in.

**Review by ghost — APPROVED** [review]
Ok for me.

**Comment by chuckwondo** [comment]
@nemo794, okay for me to merge?

**Pull request merged**
